ui.apps/src/main/content/jcr_root/apps/aem-dictionary-translator/content/granite/dialog/dictionary/label/publish/.content.xml

```diff
@@ -0,0 +1,66 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<jcr:root
```

Comment:
Cant this dialog be reused from the existing one?

Reply:
might be but in the existing one we talk about publishing a dictionary and this one will talk about labels